kolibri/core/assets/src/views/AndroidNavigationNestedMenu.vue

```diff
@@ -0,0 +1,550 @@
+<template>
```

Comment:
I know there are some differences here, but the wholesale duplication of the SideNav component with some mild tweaks feels fairly problematic here. I would much rather we either conditionalize the existing SideNav component with props, or break it up slightly to allow the display as a slide out with modal backdrop. Either way, as all the strings in here already exist in SideNav, we can reuse those and ensure strings stay in the SideNav component.